apps/perf-test/src/scenarios/CompoundButtonNext.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { CompoundButton } from '@fluentui/react-button';
+
+const Scenario = () => (
```

Comment:
i don't think perf test is needed since we already have one for button (to mainly save build time)

Reply:
Removed.